### ISSUE TITLE
Web Scrapping

### ISSUE THREAD

**Issue by rSterling319** [Shinon245/MealPlanner#1]
Start with scraping a recipe from Allrecipes.com. Expand to other popular recipe sites.

**Comment by zanoniaustin** [comment]
I'll try and start looking into this since I feel like yall are able to do databases better than me.

**Comment by rSterling319** [comment]
As we are working on the scrapper something to think about is how we will store the Recipes -- I think it will make database transactions easier if we basically just hand it a Recipe object. We program a database helper that knows how to take a recipe object and push it to the database, all we have to do is give the helper the Recipe object.
So as we are working on the scrapping side I think keep in mind we will have a Recipe object that has a list of Ingredients and a list of directions. Maybe I should make another issue to make recipe OO stuff.

**Comment by Shinon245** [comment]
Good idea, using js it shouldn't be too hard at all to create an object with all the different properties of the recipe we want. If I recall correctly, js objects work really similarly to like java or c# objects.

**Comment by rSterling319** [comment]
We are going to need a parser to parse the text of the ingredient to pull out the values of the measurement, unit and the text. For example the scraper gets 1/2 cup rice which needs to be split up into 1/2, cup, rice -- that way we can scale the recipe when we need to.
